MAINTAINERS.yml

```diff
@@ -1347,6 +1347,16 @@ Filesystems:
   labels:
     - "area: File System"
 
+"Filesystems: FatFs":
+  status: maintained
+  maintainers:
+    - ox11
+  files:
+    - subsys/fs/fat_fs.c
+    - modules/fatfs/zfs_ffsystem.c
```

Comment:
List is missing test_fat_file_reentrant.c file.

Reply:
Added it.